test/unit/argument_parser/format_parse_validators_test.cpp

```diff
@@ -825,16 +825,16 @@ TEST(validator_test, value_list_validator_success)
     // type deduction
     // --------------
     // all arithmetic types are deduced to double in order to easily allow chaining of arithmetic validators
-    EXPECT_TRUE((std::same_as<seqan3::value_list_validator<double>,
+    EXPECT_TRUE((std::same_as<seqan3::value_list_validator<int>,
```

Comment:
This seems to contradict the previous comment

Reply:
I changed it to "all arithmetic types are deduced to their common type in order to easily allow chaining of arithmetic validators"